### DISCUSSION TITLE
Props stored in cookies creating problem in switching account in NextJs SSR app

### DISCUSSION THREAD

**Discussion by keshakaneria** [vercel/community#1063] [Help] [answered]
I am facing an issue in NextJs app using SSR hosted on vercel. I am storing orgID and projectID in cookies. And using nookies to set them.
![image](https://user-images.githubusercontent.com/46588494/202370682-9c1a6de4-8777-4be9-a28e-e8a440e073f8.png)

Steps which is causing me issue:
- User logs in with account1
- Dashboard with account1's data is visible (expected)
- Logs out
- Logs back in with account2
- I still sometimes see account1's data in dashboard instead of account2's data. (not expected)
- After reloading that dashboard 2-3 times, we get the original dashboard of account2.

I am also destroying all the cookies while logging out. How should I resolve this issue? and what exactly is causing this?

**Comment by amyegan** [comment, marked as answer]
Hi @keshakaneria! The `nookies` package is not developed by Vercel. So I'm not familiar with the code and can't offer the best advice on this one. 

It sounds like the cookies are not being destroyed reliably. The [readme](https://github.com/maticzav/nookies#reference) has a note about `destroyCookies` that might be related to your problem.

> Don't forget to end your response on the server with res.send(). This might be the reason your cookie isn't removed.

If that doesn't solve your problem, I would recommend that you [open an new issue in the nookies repo](https://github.com/maticzav/nookies/issues). You should include a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) there to demonstrate the exact problem you're facing.
